src/Xablu.Walkthrough/Plugin.Xablu.Walkthrough.iOS/Containers/PantheonContainer.xib

```diff
@@ -18,18 +18,18 @@
         </placeholder>
         <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
         <view contentMode="scaleToFill" id="2">
-            <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
```

Comment:
I have opened this file with XCode to use a reference for creating the Vesta iOS container. XCode has made automatic changes to the layout (probably because I used a bit different settings then you). This doesn't break anything but I will try to revert the file.